main.go

```diff
@@ -130,10 +130,7 @@ func main() {
 		if len(seed) != 32 {
 			log.Fatalln("error: Seed should be 256bit in base64")
 		}
-		idGenerator = idgen.NewBalancedIdentityGeneratorFromSeed(seed)
```

Comment:
This file still has

```
	if *idgenAddr == "" {
		*idgenAddr = os.Getenv("HYDRA_IDGEN_ADDR")
	}
```

Reply:
I didn't remove it yet, I just removed mention of it and the vars from our k8s deployment. #137 deals with it's removal (and other cleanups I think that should follow)